### PR TITLE
ENTESB-12268 Integration uptime doesn't contain any data

### PR DESCRIPTION
**Pull request by claudio4j**
https://issues.jboss.org/browse/ENTESB-12268

### CODE REVIEW

**Comment by zregvart** [comment]
Perhaps also related to #7143, not sure if we would like to provide the time in seconds and the UI does the multiply by 1000 or the backend provides the value in miliseconds and the UI takes the value verbatim.

Any thoughts @gashcrumb?

**Comment by gashcrumb** [comment]
Consistency would be better, if the UI can just use a value without having
to modify it first, that'd be better IMO

File: app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class IntegrationDetailMetrics extends React.Component<
   > {
   public render() {
     const okMessagesCount = this.props.messages! - this.props.errors!;
-    const startAsDate = new Date(this.props.start!);
+    const startAsDate = new Date(this.props.start! * 1000);
```
Comment (zregvart) [inline app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx]: Perhaps we can do this on the server (checkout the comments on PR)

File: app/ui-react/syndesis/src/modules/integrations/pages/detail/MetricsPage.tsx [hunk 2]
```diff
@@ -105,7 +105,7 @@ export class MetricsPage extends React.Component {
                                                     parseInt(
                                                       metricsData.lastProcessed,
                                                       10
-                                                    )
+                                                    ) * 1000
```
Comment (zregvart) [inline app/ui-react/syndesis/src/modules/integrations/pages/detail/MetricsPage.tsx]: Let's do this on the server, so UI doesn't have to do `* 1000`

**Comment by gashcrumb** [comment]
The other option btw is to maybe as a convention include the time unit as part of the property name, that way anybody coming along and looking can understand what's going on without having to stare at the value or see it convert to some date in the 70s :-)

**Comment by claudio4j** [comment]
I found Zoran suggestion better, do it on server. I am testing this option and will push the commit today. Thanks for the review.

File: app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx (hunk 1 above)
Comment (claudio4j) [reply app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx]: Done, thanks.

File: app/ui-react/syndesis/src/modules/integrations/pages/detail/MetricsPage.tsx (hunk 2 above)
Comment (claudio4j) [reply app/ui-react/syndesis/src/modules/integrations/pages/detail/MetricsPage.tsx]: Done.

**Comment by claudio4j** [comment]
> Perhaps also related to #7143

This fix the above issue.

File: app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx (hunk 1 above)
Comment (claudio4j) [reply app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx]: Done.

**Pull request merged**